corehq/form_processor/migrations/0071_case_attachment_refactor.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-07-03 15:57
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.db import migrations
+from corehq.sql_db.operations import HqRunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('form_processor', '0070_auto_20180329_1016'),
+    ]
+
+    operations = [
+        migrations.AlterIndexTogether(
+            name='caseattachmentsql',
+            index_together=set([('case', 'name')]),
```

Comment:
This will lock the table when creating the index since it isn't concurrent by default. May want to see how large the table is on each env to see how long this will take. Also I don't see this dropping the index on (case, identifier)

Reply:
There are very few projects that use case attachments, and therefore the table is not very big. I just looked and it's less than 4K rows per shard.

The (case, identifier) index will be deleted when the `identifier` column is dropped by #21107